sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicSyntaxHelper.cs

```diff
@@ -81,5 +81,22 @@ public static StatementSyntax GetSucceedingStatement(this StatementSyntax curren
 
         public static bool AnyOfKind(this IEnumerable<SyntaxToken> tokens, SyntaxKind kind) =>
             tokens.Any(n => n.RawKind == (int)kind);
+
+        public static SyntaxToken? GetMethodCallIdentifier(this InvocationExpressionSyntax invocation)
+        {
+            if (invocation == null)
+            {
+                return null;
+            }
+            if (invocation.Expression is IdentifierNameSyntax directMethodCall)
+            {
+                return directMethodCall.Identifier;
+            }
+            if (invocation.Expression is MemberAccessExpressionSyntax memberAccessCall)
```

Comment:
Same as for C# here.

Reply:
in vb.net when using the null-conditional operator (e.g. `handle?.IsClosed`), the parser will generate a `SimpleMemberAccessExpression` as well (as opposed to C#)

Reply:
Please add this comment in the code directly. It will avoid wondering why we didn't handled this case.